api/valor_api/backend/core/dataset.py

```diff
@@ -417,7 +404,6 @@ def get_dataset_summary(db: Session, name: str) -> schemas.DatasetSummary:
         ),
         num_polygons=get_n_groundtruth_polygons_in_dataset(db, name),
         num_rasters=get_n_groundtruth_rasters_in_dataset(db, name),
-        task_types=get_unique_task_types_in_dataset(db, name),
```

Comment:
I still feel like this is useful b/c it shows the type of data that stored.

Reply:
data doesn't have "types" per this PR; the point was to separate the concern between datums and evaluation types. what's a practical example where this is useful?

Reply:
In the ingestion validator you have an implied data type. Why not use this?